docker/tools/Dockerfile

```diff
@@ -10,8 +10,7 @@ LABEL name="kanister-tools" \
 
 COPY --from=restic/restic:0.11.0 /usr/bin/restic /usr/local/bin/restic
 # kastenhq/kopia alpine-sha-7664c8c image
-COPY --from=ghcr.io/kastenhq/kopia@sha256:540e05631dc48428725dc938ce62d7e4c1e698ba1fc6956ad36dd349ad076a20 \
-  /kopia/kopia /usr/local/bin/kopia
+COPY --from=ghcr.io/kanisterio/kopia:alpine-317cc36 /kopia/kopia /usr/local/bin/kopia
```

Comment:
Does one have to update and commit this file each time they want to update kopia binary in tools image?

Reply:
Yes, since we want a specific version of the kopia binary in kanister-tools, we should always specify it here and update whenever needed.